kubernetes/setup-environment

```diff
@@ -0,0 +1,19 @@
+log()   { echo ">>> $1" ; }
+
+until netstat -antp | grep ":::3306" &> /dev/null;
+do
+    log "Waiting for mariadb database to be ready"
+    sleep 5
+done
+
+log "MariaDB container is ready"
+log "Creating database"
+docker exec -i $(docker ps | grep pharos-dashboard | awk '{print $1}') rake db:create &> /dev/null
+docker exec -i $(docker ps | grep pharos-dashboard | awk '{print $1}') bash -c "RAILS_ENV=test rake db:create" &> /dev/null
+log "Loading schema"
+docker exec -i $(docker ps | grep pharos-dashboard | awk '{print $1}') rake db:schema:load &> /dev/null
+docker exec -i $(docker ps | grep pharos-dashboard | awk '{print $1}') bash -c "RAILS_ENV=test rake db:schema:load" &> /dev/null
```

Comment:
Maybe we can remove the dummy Minions from seeds, add a default User there and run seeds. Right now we have to manually create a user in order to access the dashboard.